ci-account-setup/doc/features.md

```diff
@@ -0,0 +1,41 @@
+# Features of th AWS CI Account Setup
+
+The AWS CI Account Setup provisions an AWS account for running tests from CI securely.
+
+## Create AWS resources from GitHub Actions
+
+`feat~create-aws-cresources-in-ci~1`
+
+We can create AWS resources from CI builds in out GitHub Actions CI builds.
+
+Rationale:
+
+* We can not use a different CI system since [Release Droid](https://github.com/exasol/release-droid) is built using GitHub actions.
+* Even so it's favorable to use local resources like local versions in docker containers for some integrations we need to test against the real cloud services. Possibles reasons are:
+    * There is no adequate local version
+    * Performance testing
+
+Needs: dsn
+
+## Damage Minimization on Hacking Attacks
+
+`feat~damage-minimization-on-hacking-attacks~1`
+
+The AWS CI account setup implements measurements to reduce the damage (costs, abuse) in case an attacker gains access to the CI account.
```

Comment:
```suggestion
The AWS CI Account Setup implements measurements to reduce the damage (costs, abuse) in case an attacker gains access to the CI account.
```